src/content/resources/cpp/multidimensional-arrays.md

```diff
@@ -0,0 +1,63 @@
+
+### How do we make one?
+
+To create multidimensional array, you do it like a regular array but with a extra `[]` at the end. 
+
+#### Method 1: 
+Use `data_type array_name[A][B]` where `data_type` is an data type(such as `int`, `char` or etc.), `array_name` is what the name of the array will be and `A` and `B` are the integers.
+
+Example: `int arr[2][3]` will make a empty 2x3 matrix that looks like: `{{0,0,0},{0,0,0}}`
+
+#### Method 2:
+Use `data_type array_name[][] = {{a1,a2,a3,...},{b1,b2,b3,...}}` where `data_type` is an data type(such as `int`, `char` or etc.), `array_name` is what the name of the array will be and `a1, a2, a3, b1, b2, b3,...` are objects of the data type that was chosen at the start in `data_type`, but the number of `a`'s and `b`'s must be equal.
+
+Example: `int arr[][] = {{1,2,3}, {4,5,6}}` will make a 2x3 matrix that looks like: `{{1,2,3},{4,5,6}}`
+
+#### Method 3:
+
+Combine both `data_type array_name[A][B] = {{a1,a2,a3,...},{b1,b2,b3,...}}` where `data_type` is an data type(such as `int`, `char` or etc.), `array_name` is what the name of the array will be and `a1, a2, a3, b1, b2, b3,...` are objects of the data type that was chosen at the start in `data_type` and `A` and `B` is the number of `{}`'s and `a`'s and `b`'s respectively.
+
+Example: `int arr[2][3] = {{1,2,3}, {4,5,6}}` will make a 2x3 matrix that looks like: `{{1,2,3},{4,5,6}}`.
+
+#### Note:
+It is possible to go ever further, and so a declaration like `data_type array_name[A][B][C]` is possible and would add "height" to your array. 
+
+For example, if you do: `int arr[2][3][4]`, you'll make an array that looks like: `{{{0,0,0,0},{0,0,0,0},{0,0,0,0}},{{0,0,0,0},{0,0,0,0},{0,0,0,0}}}`
+
+### What can you do with it?
+
+Everything you can do with a one dimensional array.
+
+#### Calling objects
+To get the object inside, you can call for the object inside a multidimensional array by putting a integer inside the `[]`. 
+So let's take for example the array made in method 2, which looks like: `int arr[2][3] = {{1,2,3}, {4,5,6}}`
+`std::cout << arr[0][1]` would output `2` because it's is the one in position 1 of the 0th array.
+
+You must put an integer in both `[]`, or else you will get an error. 
+There is no way syntax to instantly view the whole array, but an easy way is to loop through the whole array and outputting it.  
+
+#### Replacing objects
+Like with a regular array, you can also change what's inside using `=`
+So let's take for example the array made in method 2, which looks like: `int arr[2][3] = {{1,2,3}, {4,5,6}}`
+`arr[1][1] = 1337` would cause the array to change to looking like `{{1,2,3},{4,1337,6}}`
```

Comment:
Maybe say you can "reassign" values inside the array

Reply:
Okay, I changed it to match that.